src/pe/header.rs

```diff
@@ -29,6 +80,31 @@ impl DosHeader {
                 signature
             )));
         }
+
+        let bytes_on_last_page = bytes.gread_with(&mut offset, scroll::LE)?;
+        let pages_in_file = bytes.gread_with(&mut offset, scroll::LE)?;
+        let relocations = bytes.gread_with(&mut offset, scroll::LE)?;
+        let size_of_header_in_paragraphs = bytes.gread_with(&mut offset, scroll::LE)?;
+        let minimum_extra_paragraphs_needed = bytes.gread_with(&mut offset, scroll::LE)?;
+        let maximum_extra_paragraphs_needed = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_relative_ss = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_sp = bytes.gread_with(&mut offset, scroll::LE)?;
+        let checksum = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_ip = bytes.gread_with(&mut offset, scroll::LE)?;
+        let initial_relative_cs = bytes.gread_with(&mut offset, scroll::LE)?;
+        let file_address_of_relocation_table = bytes.gread_with(&mut offset, scroll::LE)?;
+        let overlay_number = bytes.gread_with(&mut offset, scroll::LE)?;
+        let reserved = [0x0; 4];
+        offset += core::mem::size_of::<[u16; 4]>();
```

Comment:
```suggestion
        offset += core::mem::size_of_val(&reserved);
```

Reply:
See nightly remark on the other comment.